src/components/subsidy-request-configuration/SubsidyRequestConfigurationContextProvider.jsx

```diff
@@ -23,13 +23,45 @@ const SubsidyRequestConfigurationContextProvider = ({ enterpriseUUID, children }
   );
 };
 
+const SubsidyRequestConfigurationContextProviderWrapper = ({ enableBrowseAndRequest, ...props }) => {
+  if (enableBrowseAndRequest) {
+    return <SubsidyRequestConfigurationContextProvider {...props} />;
```

Comment:
nit: hmm, might just be personal preference so feel free to disregard, but I think it might be good to be explicit about which props we're passing here to alleviate (my) confusion:

```jsx
const SubsidyRequestConfigurationContextProviderWrapper = ({ enableBrowseAndRequest, enterpriseUUID, children }) => {
  if (enableBrowseAndRequest) {
    return (
      <SubsidyRequestConfigurationContextProvider enterpriseUUID={enterpriseUUID}>
        {children}
      </SubsidyRequestConfigurationContextProvider>
    );
  }

  const context = { subsidyRequestConfiguration: { enterpriseCustomerUuid: enterpriseUUID, ... } };
  return (
    <SubsidyRequestConfigurationContext.Provider value={context}>
      {children}
    </SubsidyRequestConfigurationContext.Provider>
  );
}
```